_source/_posts/2024-08-15-otp-over-sms.md

```diff
@@ -0,0 +1,178 @@
+---
+layout: blog_post
+title: "Approaches to keep sending OTP over SMS... for now"
+author: maurice-sharp
+by: contractor
+communities: [devops,.net]
+description: "Learn how to make the best telephony choices for your application"
+tags: [devops]
+tweets:
+- ""
+image: blog/telephony/social.jpg
+type: conversion
+---
+{% include toc.md %}
+
+## Approaches to keep sending OTP over SMS... for now
```

Comment:
```suggestion
```

Please delete this header. This is the same as the title. A blog should not start immediately with a header as the title is a header. There needs to be an intro.